sdk/core/azure-core/src/test/java/com/azure/core/credential/TokenCacheTests.java

```diff
@@ -71,7 +71,7 @@ public void testLongRunningWontOverflow() throws Exception {
 
         virtualTimeScheduler.advanceTimeBy(Duration.ofSeconds(40));
 
-        assertTrue(latch.await(30, TimeUnit.SECONDS));
+        assertTrue(latch.await(60, TimeUnit.SECONDS));
```

Comment:
there used to be no explicit timeout here and it turns out this test could run for [40+ seconds](https://dev.azure.com/azure-sdk/public/_build/results?buildId=2741645&view=ms.vss-test-web.build-test-results-tab&runId=40672150&resultId=104412) and finish successfully. So increasing timeout to stabilize test.